client/src/test/diagnostics.test.ts

```diff
@@ -1,56 +0,0 @@
-/* --------------------------------------------------------------------------------------------
```

Comment:
This code might be useful to bring back. But it is reasonable to start with a cleanup and only bring back what we need 👍

Reply:
Hmm, but on a second thought. It seems we would need most of these files? Or is that not the case?

Reply:
All of it should be available from here: https://code.visualstudio.com/api/working-with-extensions/testing-extension
I think we should just get rid of it and include the necessary.

Reply:
But would it not be better to see if you can get this test file to run? and then delete everything not related to that-- instead of startin from scratch?

Reply:
Most of it appears to be very specific to the example extension